Privacy Policy.md

```diff
@@ -0,0 +1,59 @@
+# WHATWG Website Privacy Policy
+
+Last updated April 20, 2020
+
+WHATWG respects your privacy and is committed to protecting your personal data. This privacy policy covers how we collect, use, and disclose your personal information when you visit our website, [whatwg.org](https://whatwg.org/). We make no effort to identify public users of our site. No identifying data is disclosed to any third party for any purpose. Data that we collect is used only for server administration, site improvement, usage statistics, and Web protocol research.
```

Comment:
```suggestion
WHATWG respects your privacy and is committed to protecting your personal data. This privacy policy covers how we collect, use, and disclose your personal information when you visit our website, [whatwg.org](https://whatwg.org/) and subdomains. We make no effort to identify public users of our site. No identifying data is disclosed to any third party for any purpose. Data that we collect is used only for server administration, site improvement, usage statistics, and Web protocol research.
```

Reply:
Suggestion came from @domenic in https://github.com/whatwg/sg/pull/120#discussion_r416842338